README.md

```diff
@@ -1,10 +1,10 @@
 Rison Parser and Generator for Jackson
 ======================================
 
-A plugin for the [Jackson streaming JSON processor v2.9.x] (http://wiki.fasterxml.com/JacksonHome) that adds
```

Comment:
Changes like these don't really belong in this patch. Please remove it and move it to a different PR if we still need it.

Reply:
it just fixes broken links

Reply:
I would rather it be in a separate PR, just as a matter process. PR's are nearly free, and we can roll all changes into a single release.

Reply:
ok, will revert that